packages/lib/src/vis/tiles/Tile.tsx

```diff
@@ -0,0 +1,35 @@
+import { NearestFilter, Vector2 } from 'three';
+import type { TextureFilter } from 'three';
+
+import HeatmapMesh from '../heatmap/HeatmapMesh';
+import { useAxisSystemContext } from '../shared/AxisSystemContext';
+import type { TilesApi } from './api';
+import type { ColorMapAttributes } from './models';
+
+interface Props extends ColorMapAttributes {
+  api: TilesApi;
+  lod: number;
+  x: number;
+  y: number;
+  magFilter?: TextureFilter;
```

Comment:
As you specify `magFilter` for each call of `Tile`, I would not bother to make it an optional with a default value that is never used.